### ISSUE TITLE
Track and show progress during region update

### ISSUE THREAD

**Issue by paetz** [YacGroup/yacguide#106]
Instead of a simple "Lädt..." information during sector updates, it would be nice to have a progress bar or sth. similar that shows the update's progress

**Comment by paetz** [comment]
Blocked by #131.

If #131 resolves this problem, too, the progress update may be only necessary for #134
